pkgs/applications/audio/ardour/ffmpeg_harvid.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, ffmpeg }:
+
+stdenv.mkDerivation {
+  name = "ffmpeg_harvid";
+
+  buildInputs = [ ffmpeg ];
+
+  phases = [ "installPhase" ];
+
+  installPhase = ''
+    mkdir -p $out/bin/
+    ln -s ${stdenv.lib.makeBinPath [ ffmpeg ]}/ffmpeg $out/bin/ffmpeg_harvid
+    ln -s ${stdenv.lib.makeBinPath [ ffmpeg ]}/ffprobe $out/bin/ffprobe_harvid
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A link to ffmpeg 2 under a different name, needed for Ardour format conversion";
+    longDescription = ''
+      Two binaries called ffmpeg_harvid and ffprobe_harvid are part of http://x42.github.io/harvid/,
+      which Ardpour uses for it's video timeline, but also some other file format conversions.
+      If you don't need the video timeline, this pkg will enable format conversion.
```

Comment:
Reading http://jlhconsulting.gotdns.com/video-timeline/setup/, it seems like the reason for rename is so that the binary releases of ffmpeg from Ardour team don't overlap with the systemwide ffmpeg. This should not be an issue with nix, we could just do
```
substituteInPlace libs/ardour/video_tools_paths.cc \
  --replace 'ffmpeg_exe = X_("");' 'ffmpeg_exe = X_("${ffmpeg}/bin/ffmpeg");' \
  --replace 'ffprobe_exe = X_("");' 'ffprobe_exe = X_("${ffmpeg}/bin/ffprobe");'
```

Reply:
I added ffmpeg to enable mp3 import and export.
Is this the right file to patch?

Reply:
ping @x42

Reply:
`grep -r ffmpeg_harvid /nix/store/16y7dp5yzy2szf6s3a6lh9rybvgja4c5-ardour` seems to not find any other places

Reply:
Works like a charm!